src/features/domain-preview/DomainPreview.tsx

```diff
@@ -26,36 +32,56 @@ export const DomainPreview: FC<DomainPreviewProps> = ({
 	creator,
 	isNFTView,
 }) => {
+	const members = [
+		{ title: MemberTitle.CREATOR, address: creator },
+		{ title: MemberTitle.OWNER, address: owner },
+	];
+
 	return (
-		<div style={{ margin: '16px 0' }}>
-			{title && <h1>{title}</h1>}
-
-			{isNFTView && creator && owner && (
-				<ul style={{ display: 'flex', padding: '0', listStyle: 'none' }}>
-					<li style={{ display: 'flex', flexDirection: 'column' }}>
-						<span>{AddressTitle.CREATOR}</span>
-						<span>{truncateAddress(creator)}</span>
-					</li>
-					<li
-						style={{
-							display: 'flex',
-							marginLeft: '32px',
-							flexDirection: 'column',
-						}}
-					>
-						<span>{AddressTitle.OWNER}</span>
-						<span>{truncateAddress(owner)}</span>
-					</li>
-				</ul>
-			)}
-
-			{description && <p>{description}</p>}
-
-			{href && (
-				<Link style={{ background: 'none', color: '#52cbff' }} to={href}>
-					{'View Domain NFT ->'}
-				</Link>
-			)}
+		<div className={styles.Container}>
+			{/* todo: media asset component */}
```

Comment:
comments added to reflect these are just placeholder components for now to help with styling